### PR TITLE
feat: add new medianizer composite adapter

### PR DESCRIPTION
**Pull request by boxhock**

### CODE REVIEW

File: packages/tsconfig.json [hunk 1]
```diff
@@ -245,6 +248,9 @@
     {
       "path": "./sources/kaiko"
     },
+    {
```
Comment (cds95) [inline packages/tsconfig.json:251]: Is this intentional?

**Review by cds95 — COMMENTED** [review]
This looks good to me.  Are there any tests for this adapter?

File: packages/tsconfig.json (hunk 1 above)
Comment (boxhock) [reply packages/tsconfig.json:251]: Yeah, it was accidentally not included in the original PR, so we should bring it in ASAP

File: packages/composites/medianizer/test/integration/adapter.test.ts [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+describe('medianizer', () => {
+  let execute: Execute
+  let config: Config
+  let sandbox: sinon.SinonSandbox
+  let server: sinon.SinonFakeServer
+
+  beforeEach(() => {
+    execute = makeExecute()
+    config = makeConfig()
+    sandbox = createSandbox()
```
Comment (HenryNguyen5) [inline packages/composites/medianizer/test/integration/adapter.test.ts:44]: Can we use the same testing pattern that's presented in `packages/composites/synth-index/test/integration`? We use nock which lets you record http calls first, which makes it super easy to generate stubs later on

**Review by HenryNguyen5 — CHANGES_REQUESTED** [review]

File: packages/composites/medianizer/test/integration/adapter.test.ts (hunk 2 above)
Comment (boxhock) [reply packages/composites/medianizer/test/integration/adapter.test.ts:44]: Happy to do this in the test improvements of `outlier-detection`, though I don't see it as necessary to get this PR out.

**Review by HenryNguyen5 — APPROVED** [review]

**Pull request merged**